packages/components/src/lib/distributionSpecBuilder.ts

```diff
@@ -13,9 +13,9 @@ export type DistributionChartSpecOptions = {
   /** The title of the chart */
   title?: string;
   /** The formatting of the ticks */
-  format?: string;
+  format: string;
```

Comment:
I think we should prefer to have props not optional lower down, to encourage setting defaults in the same place.